apps/backoffice-v2/src/pages/Entity/Entity.page.tsx

```diff
@@ -23,13 +21,6 @@ export const Entity = () => {
         workflow={workflow as TWorkflowById}
       />
       <Case.Content key={selectedEntity?.id}>
-        {workflow?.workflowDefinition?.config?.isCaseOverviewEnabled && (
-          <ProcessTracker
-            plugins={plugins}
-            workflow={workflow}
-            processes={processTrackerProcesses}
-          />
-        )}
```

Comment:
This is an instance that the tabs render the process tracker and in the instances where the process tracker was added are places with no tabs?